app/src/main/java/io/github/wulkanowy/data/db/entities/Recipient.kt

```diff
@@ -0,0 +1,40 @@
+package io.github.wulkanowy.data.db.entities
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.io.Serializable
+
+@Entity(tableName = "Recipients")
+data class Recipient(
+
+    @ColumnInfo(name = "student_id")
+    val studentId: Int,
+
+    @ColumnInfo(name = "real_id")
+    val realId: String,
+
+    val name: String,
+
+    @ColumnInfo(name = "real_name")
+    val realName: String,
+
+    @ColumnInfo(name = "login_id")
+    val loginId: Int,
+
+    @ColumnInfo(name = "unit_id")
+    val unitId: Int,
+
+    val role: Int,
+
+    val hash: String
+
+) : Serializable {
+
+    @PrimaryKey(autoGenerate = true)
+    var id: Long = 0
+
+    override fun toString(): String {
```

Comment:
https://github.com/wulkanowy/wulkanowy/blob/c46a9b5563f0e89f4e86e0a8359170d034ce14c8/app/src/main/java/io/github/wulkanowy/data/db/entities/Recipient.kt#L37-L39

```suggestion
    override fun toString() = name
```